dev-python/nose_warnings_filters/nose_warnings_filters-0.1.5.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="A python module to inject warning filters during nosetest"
+HOMEPAGE="https://github.com/Carreau/nose_warnings_filters"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${PYTHON_DEPS}
+	dev-python/nose[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+src_test() {
+	# nose_warnings_filters doesn't have a proper
+	# testing suite, hence we run the only testing
+	# script available
+	export PYTHONPATH="${S}:${S}/${PN}/tests"
+	run_test() {
+		"${PYTHON}" "${PN}"/testing/test_config.py \
+			|| die "Failed running test script"
+	}
+	python_foreach_impl run_test
+	unset PYTHONPATH
```

Comment:
@marbre yikes, that's embarrassing :hushed:
merged, it should be properly fixed now, have a look if you like